core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -272,17 +281,25 @@ class AdminApiServiceSpec extends ApiServiceSpec {
   it should "return 404 when removing a nonexistent user from a billing project" in withTestDataApiServices { services =>
     val project = billingProjectFromName("new_project")
 
-    Put(s"/admin/billing/register/${project.projectName.value}") ~>
-      sealRoute(services.adminRoutes) ~>
-      check {
-        Delete(s"/admin/billing/${project.projectName.value}/nobody") ~>
-          sealRoute(services.adminRoutes) ~>
-          check {
-            assertResult(StatusCodes.NotFound) {
-              status
+    val putUri: Uri = s"/admin/billing/register/${project.projectName.value}"
+    val delUri: Uri = s"/admin/billing/${project.projectName.value}/nobody"
+
+    val sealedInstrumentedAdminRoutes: Route = sealRoute(instrumentRequest { services.adminRoutes })
```

Comment:
I'm thinking about adding something like this, maybe to `ApiServices`:

`val allRoutesInstrumented = sealRoute(instrumentRequest {
  services.adminRoutes ~ services.billingRoutes   // etc
}`

This would be similar to what we're doing in `RawlsApiServiceActor`.